cypress/helpers/localStorage.js

```diff
@@ -0,0 +1,38 @@
+import "cypress-localstorage-commands";
+
+export function setSliceMachineUserContext({
+  hasSendAReview, // boolean
+  isOnboarded, // boolean
+  updatesViewed, // object
+  hasSeenTutorialsTooTip, // boolean
+}) {
```

Comment:
Can we make these all optional with defaults so we only overload the field we want to change? Then spread it below

Reply:
I personally prefer to state all the values as I think it makes it clearer to understand from the test itself what happens.
But you and previously Hugo seems to agree that it would be better with default values 👍.

I'll modify it 😄